src/sora-academy.md

```diff
@@ -1,3 +1,53 @@
 # SORA Academy
 
-The [SORAcademy](https://medium.com/sora-academy) provides tutorial on how to use SORA.
+Thanks to blockchain technology, finance has become more inclusive and
+less centralized, sparking a sector known as Decentralized Finance,
+and with access to a crypto asset wallet, people can participate in
+activities such as swapping tokens (exchanges), liquidity provision,
+NFT Markets, and more. The SORA Academy is the best way to get you started in DeFi on the SORA network
+
+## Decentralization and Finance
+
+When you participate in any financial activity nowadays, such as investing in stocks or bonds, there are several hurdles you must jump through in order to begin trading, your identity must be verified and you must open an account with an intermediary service to trade. Albeit, some of those services provide the option to pool with other users and buy fractions of stocks, instead of the whole stock, as was the norm up until these services were developed.
```

Comment:
```suggestion
When you participate in any financial activity nowadays, such as investing in stocks or bonds, there are several hurdles you must jump through in order to begin trading: your identity must be verified and you must open an account with an intermediary service to trade. Albeit, some of those services provide the option to pool with other users and buy fractions of stocks, instead of the whole stock, as was the norm up until these services were developed.
```